CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 1.6.1.3
```

Comment:
I'm not sure why did you decide to include two versions in the changelog ?
With your PR `v1.6.1.2` would never be released, so there is no point including it in the changelog.

Reply:
Right - dropping this entry and merging with the other changes. Also adjusting change log towards version '1.7.0'.